src/elements/netspeak-search.tsx

```diff
@@ -360,8 +376,10 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
 		this._setExampleVisibility(this._areExamplesVisible() ? "hidden" : "visible");
 	};
 	private _onClearButtonClick = (): void => {
-		this._setQuery("", true);
-		this._clearPhrases(true);
+		if (!this.props.showExperimental) {
```

Comment:
This `if` is useless because in `showExperimental` mode the clear button isn't present, so it can't be clicked.

Same for these ifs in `_onExampleQueryClickHandler`, `_onSearchBarQueryEnterHandler`, and `_onSearchBarQueryEnterHandler`.